src/development/building4diac.asciidoc

```diff
@@ -20,32 +20,46 @@ Run through the following steps to build and execute the 4diac IDE from source:
 
 === [[devEnvironment]]Get the Development Environment
 
+The 4diac IDE requires at least a version 17 Java JRE. Higher version JREs are supported, but may require additional Eclipse IDE configuration (see step 4 of link:#importPlugins[Import plug-ins into workspace]).
```

Comment:
We follow the common AsciiDoc practice to have one line per sentence. Can you please change your paragraphs to this policy.

Reply:
Fixed - Policy applied across the file!

Reply:
thx for fixing.